phpBB/styles/prosilver/theme/mentions.css

```diff
@@ -0,0 +1,86 @@
+/* -------------------------------------------------------------- /*
+	$Mentions
+/* -------------------------------------------------------------- */
+
+/* stylelint-disable selector-max-compound-selectors */
+/* stylelint-disable selector-no-qualifying-type */
+
+/* Mention block
+---------------------------------------- */
+.mention {
+	font-weight: bold;
+}
+
+/* Mention dropdown
+---------------------------------------- */
+.atwho-view { /* mention-container */
+	text-align: left;
+	border-radius: 2px;
+	position: absolute;
+	z-index: 999;
+	overflow: auto;
+	transition: all 0.2s ease;
+}
+
+.atwho-view-ul { /* mention-list */
+	margin: 0;
+	padding: 0;
+	list-style-type: none;
+}
+
+.mention-media {
+	display: inline-flex;
+	flex-shrink: 0;
+	justify-content: center;
+	align-items: center;
+	margin-right: 16px;
+	margin-left: 0;
+}
+
+.mention-media-avatar {
+	width: 40px;
+	height: 40px;
+}
+
+svg.mention-media-avatar { /* TODO: remove it after general normalization */
```

Comment:
not sure why you are class typing? This is a global for all svg...leave the comment for me to clean up but remove the class it should just be `svg {`

Reply:
Added that not to make a global impact in the scope of this PR.
Ok, I can make it global if you say so.

Reply:
Yes it’s a global it needs to be in the reset